reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md

```diff
@@ -15,116 +15,152 @@ Sends ICMP echo request packets ("pings") to one or more computers.
 
 ### Default (Default)
 ```
-Test-Connection [-AsJob] [-Authentication <AuthenticationLevel>] [-BufferSize <Int32>]
- [-ComputerName] <String[]> [-Count <Int32>] [-Impersonation <ImpersonationLevel>] [-ThrottleLimit <Int32>]
- [-TimeToLive <Int32>] [-Delay <Int32>] [<CommonParameters>]
+Test-Connection [-ComputerName] <string[]> [-AsJob] [-Authentication <AuthenticationLevel>] [-BufferSize <int>]
+  [-Count <int>] [-Impersonation <ImpersonationLevel>] [-ThrottleLimit <int>] [-TimeToLive <int>]
+  [-Delay <int>] [<CommonParameters>]
 ```
 
 ### Source
 ```
-Test-Connection [-AsJob] [-Authentication <AuthenticationLevel>] [-BufferSize <Int32>]
- [-ComputerName] <String[]> [-Count <Int32>] [-Credential <PSCredential>] [-Source] <String[]>
- [-Impersonation <ImpersonationLevel>] [-ThrottleLimit <Int32>] [-TimeToLive <Int32>] [-Delay <Int32>]
- [<CommonParameters>]
+Test-Connection [-ComputerName] <string[]> [-Source] <string[]> [-AsJob] [-Authentication <AuthenticationLevel>]
+  [-BufferSize <int>] [-Count <int>] [-Credential <pscredential>] [-Impersonation <ImpersonationLevel>]
+  [-ThrottleLimit <int>] [-TimeToLive <int>] [-Delay <int>] [<CommonParameters>]
 ```
 
 ### Quiet
 ```
-Test-Connection [-Authentication <AuthenticationLevel>] [-BufferSize <Int32>] [-ComputerName] <String[]>
- [-Count <Int32>] [-Impersonation <ImpersonationLevel>] [-TimeToLive <Int32>] [-Delay <Int32>] [-Quiet]
- [<CommonParameters>]
+Test-Connection [-ComputerName] <string[]> [-Authentication <AuthenticationLevel>] [-BufferSize <int>]
+  [-Count <int>] [-Impersonation <ImpersonationLevel>] [-TimeToLive <int>] [-Delay <int>] [-Quiet]
+  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The Test-Connection cmdlet sends Internet Control Message Protocol (ICMP) echo request packets ("pings") to one or more remote computers and returns the echo response replies.
-You can use this cmdlet to determine whether a particular computer can be contacted across an Internet Protocol (IP) network.
 
-You can use the parameters of Test-Connection to specify both the sending and receiving computers, to run the command as a background job, to set a timeout and number of pings, and to configure the connection and authentication.
+The **Test-Connection** cmdlet sends Internet Control Message Protocol (ICMP) echo request packets,
```

Comment:
This and the next Test-Connection are in bold, then the third is code-fenced.  The contrib guide doesn't say what it should be after the first instance... but it should be consistent.

Reply:
Fixed